client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -39,14 +44,37 @@ export class SimpleSiteRenameForm extends Component {
 		this.props.requestSiteRename( selectedSiteId, this.state.domainFieldValue, discard );
 	};
 
+	getDomainValidationMessage( domain ) {
+		const { translate } = this.props;
+
+		if ( ! domain.match( /^[a-z0-9]+$/i ) ) {
+			return translate( 'Domain can only contain letters and numbers' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Usernames can only contain lowercase letters (a-z) and numbers.' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/sites-rename--validation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).